### DISCUSSION TITLE
Emit CS8785 as an error

### DISCUSSION THREAD

**Discussion by StijnOostdam** [dotnet/roslyn#58900] [Q&A]
As a continuation of my [previous question](https://github.com/dotnet/roslyn/discussions/58862), is it possible to promote CS8785 (and CS8784) to an error? Such that my build fails if something is configured wrongly. I have tried rulesets and `<WarningsAsErrors>` but it seems that those are ignored since CS8785 is a CSC error.

**Comment by Youssef1313** [comment]
Have you tried setting `<TreatWarningsAsErrors>true</TreatWarningsAsErrors>` as well?

**Comment by PathogenDavid** [comment]
This is a known issue: https://github.com/dotnet/roslyn/issues/49219 https://github.com/dotnet/roslyn/issues/46939#issuecomment-725060130
